modules/007-registrypackages/images/kubernetes-cni-altlinux/werf.inc.yaml

```diff
@@ -0,0 +1,54 @@
+{{- $cni_versions := list }}
+{{- range $key, $value := .CandiVersionMap.k8s }}
+  {{- $cni_versions = $value.cniVersion | toString | append $cni_versions }}
+{{- end }}
+{{- range $value := $cni_versions | uniq }}
+  {{- $version := $value | toString }}
+  {{- $image_version := $version | replace "." "-" }}
+---
+image: {{ $.ModuleName }}/{{ $.ImageName }}-{{ $image_version }}
+from: {{ $.Images.BASE_SCRATCH }}
+import:
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-artifact-{{ $image_version }}
+  add: /
+  to: /
+  includePaths:
+  - install
+  - uninstall
+  - bandwidth
+  - bridge
+  - dhcp
+  - firewall
+  - host-device
+  - host-local
+  - ipvlan
+  - loopback
+  - macvlan
+  - portmap
+  - ptp
+  - sbr
+  - static
+  - tuning
+  - vlan
+  - vrf
+  before: setup
```

Comment:
i cannot see flannel binary.

Reply:
flannel is now in a different project, added it to the build
https://github.com/flannel-io/cni-plugin